ogr/parsing.py

```diff
@@ -136,16 +136,21 @@ def parse_git_repo(potential_url: str) -> Optional[RepoUrl]:
         )
     if not username and len(split) >= 2:
         # path contains username/reponame
+        # or some/namespace/reponame
+        # or fork/username/some/namespace/reponame
 
-        is_fork = "fork" in split
-        if is_fork and len(split) == 4:
+        is_pagure_fork = "fork" in split
```

Comment:
Would checking for `split[0] == fork` be better?

Reply:
Good point.

Reply:
Fixed.

Reply:
How about adding a test for "https://host.name/namespace/fork"? A comment saying `a repo called "fork"` would be probably useful.